packages/api/lib/ems.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+
+const { get } = require('lodash');
```

Comment:
To cut down on package size, @scisco has been encouraging us to use the "lodash.get" package from npm instead of the full lodash package.

Reply:
Looking closer, I don't see lodash listed in packages/api/package.json

```
cumulus CUMULUS-535-emsreport % grep lodash packages/api/package.json
    "lodash.clonedeep": "^4.5.0",
    "lodash.get": "^4.4.2",
    "lodash.has": "^4.5.2",
    "lodash.merge": "^4.5.0",
    "lodash.omit": "^4.5.0",
    "lodash.uniqby": "^4.7.0",
```